test/integration/api/images.bats

```diff
@@ -43,3 +43,16 @@ function teardown() {
 	run docker_swarm images -a
 	[ "${#lines[@]}" -ge 3 ]
 }
+
+@test "docker images -f label" {
+	start_docker_with_busybox 2
+	swarm_manage
+
+	docker_swarm pull dnephin/swarm-integration-test-image-filter-labels:latest
```

Comment:
Can we get away with no pull? The problem is that it slows down the integration tests - even pulling an empty image is slow because of the back and forth with the registry, the extraction, and so on.

May I suggest to do a build instead and place the `Dockerfile` somewhere in `test/integration/testdata`, just like `test/integration/api/build.bats` is doing?

Thanks!